PKHeX/MainWindow/Main.cs

```diff
@@ -840,11 +842,77 @@ private void loadSAV(SaveFile sav, string path)
             TB_AbilityNumber.Visible = SAV.Generation >= 6 && DEV_Ability.Enabled;
             CB_Ability.Visible = !DEV_Ability.Enabled && SAV.Generation >= 3;
 
+            GB_ExtraBytes.Visible = SAV.Generation > 1;
+            GB_Markings.Visible = SAV.Generation > 2;
+
+            Label_HeldItem.Visible = CB_HeldItem.Visible = SAV.Generation > 1;
+            Label_Total.Visible = TB_IVTotal.Visible = TB_EVTotal.Visible = L_Potential.Visible =
+                Label_HiddenPowerPrefix.Visible = CB_HPType.Visible = SAV.Generation > 1;
+            Label_CharacteristicPrefix.Visible = L_Characteristic.Visible = SAV.Generation > 1;
+            Label_ContestStats.Visible = Label_Cool.Visible = Label_Tough.Visible = Label_Smart.Visible =
+                Label_Sheen.Visible = Label_Beauty.Visible = Label_Cute.Visible = TB_Cool.Visible = TB_Tough.Visible =
+                    TB_Smart.Visible = TB_Sheen.Visible = TB_Beauty.Visible = TB_Cute.Visible = Label_Nature.Visible =
+                    CB_Nature.Visible = Label_Language.Visible = CB_Language.Visible = Label_Ability.Visible = 
+                    Label_Friendship.Visible = Label_HatchCounter.Visible = TB_Friendship.Visible = BTN_RerollPID.Visible = 
+                    Label_PID.Visible = TB_PID.Visible = Label_SID.Visible = TB_SID.Visible = SAV.Generation >= 3;
+
+            // Met Tab
+            CHK_Fateful.Visible = Label_OriginGame.Visible = Label_Ball.Visible = Label_MetLevel.Visible =
+                Label_MetLocation.Visible = CB_GameOrigin.Visible = CB_Ball.Visible = CB_MetLocation.Visible =
+                    TB_MetLevel.Visible = SAV.Generation > 2;
+
+            CHK_Infected.Visible = CHK_Cured.Visible = SAV.Generation >= 3;
+
+            // Second daycare slot
+            SlotPictureBoxes[43].Visible = SAV.Generation > 1;
+
+            CHK_IsEgg.Visible = Label_Gender.Visible = SAV.Generation > 1;
+
+            Label_OTGender.Visible = SAV.Generation > 1;
+
+            if (1 <= sav.Generation && sav.Generation <= 2)
+            {
+                Label_SPD.Visible = TB_SPDEV.Visible = TB_SPDIV.Visible = Stat_SPD.Visible = false;
+                Label_SPA.Text = "Spc";
```

Comment:
Changing language breaks this, recommend adding a new Label that toggles visibility.

Not sure on the positioning; for different operating system scaling modes the defined point values may cause controls to be sized incorrectly... had this issue back when PKHeX had a toggle for box interface. Maybe add extra maskedtextboxes that are also not visible.

I'll probably put all the controls into a TableLayoutPanel that'll handle the re-positioning.